.prettierrc.yml

```diff
@@ -0,0 +1,10 @@
+trailingComma: all
+printWidth: 120
+singleQuote: true
+arrowParens: avoid
+tabWidth: 4
```

Comment:
@apmorton gave some good reasons as to why we might want this to be 2 instead of 4 over at Discord some time ago

Reply:
I missremembered. Actually, they argued for tabs instead of spaces and linked to https://dev.to/alexandersandberg/why-we-should-default-to-tabs-instead-of-spaces-for-an-accessible-first-environment-101f

Reply:
I would prefer to stick to whatever we have now in the current source. I think that's 4, so this is fine.

Reply:
Yeah for now let's stick. @apmorton 's argument for tabs was compelling but ...